public/app/features/dashboard-scene/sharing/ExportButton/ExportAsJson.tsx

```diff
@@ -113,11 +113,14 @@ function ExportAsJsonRenderer({ model }: SceneComponentProps<ExportAsJson>) {
 
 function getStyles(theme: GrafanaTheme2) {
   return {
+    container: css({
+      height: '100%',
+    }),
     codeEditorBox: css({
-      margin: `${theme.spacing(2)} 0`,
+      margin: `${theme.spacing(2, 0)}`,
```

Comment:
Nonblocking question: what is the difference between this and the previous one?

Reply:
it's the same but, as we're already using `theme.spacing`, why not use it for the whole margin value 😅😅